src/AccountManager.cpp

```diff
@@ -517,21 +517,21 @@ QString AccountManager::evernoteAccountTypeToString(const Account::EvernoteAccou
 
 void AccountManager::readComplementaryAccountInfo(Account & account)
 {
-    QNDEBUG(QStringLiteral("AccountManager::readComplementaryAccountInfo: ") << account);
+    QNTRACE(QStringLiteral("AccountManager::readComplementaryAccountInfo: ") << account.name());
 
     if (Q_UNLIKELY(account.isEmpty())) {
-        QNDEBUG(QStringLiteral("The account is empty"));
+        QNDEBUG(QStringLiteral("The account is empty ") << account.name());
```

Comment:
It would be good to put a colon before the account name, just to match the style of existing log entries.